stats/bookkeeper-stats-providers/prometheus-metrics-provider/src/main/java/org/apache/bookkeeper/stats/prometheus/PrometheusMetricsProvider.java

```diff
@@ -211,12 +214,25 @@ private void registerMetrics(Collector collector) {
         }
     }
 
-
     private static final Logger log = LoggerFactory.getLogger(PrometheusMetricsProvider.class);
 
+    /*
+     * Try to get Netty counter of used direct memory. This will be correct, unlike the JVM values.
+     */
+    private static final AtomicLong directMemoryUsage;
     private static final Optional<BufferPoolMXBean> poolMxBeanOp;
 
     static {
+        AtomicLong tmpDirectMemoryUsage = null;
```

Comment:
Do we need to get direct memory usage here according to `PlatformDependent.useDirectBufferNoCleaner()` here instead of getting both values?

Reply:
done